lib/cylc/task_job_mgr.py

```diff
@@ -798,7 +797,6 @@ def _prep_submit_task_job_error(self, suite, itask, dry_run, action, exc):
         """Helper for self._prep_submit_task_job. On error."""
         LOG.debug("submit_num %s" % itask.submit_num)
         LOG.debug(traceback.format_exc())
-        LOG.error(exc)
```

Comment:
Redundant as `exc` is passed to `log_task_job_activity()` below which calls `LOG.error()` with the `SubProcContext` anyway

Reply:
Yep - just realized on closer reading that this is the case